groclient/lib.py

```diff
@@ -403,20 +407,35 @@ def get_source_ranking(access_token, api_host, series):
     return get_data(url, headers, params).json()
 
 
-def rank_series_by_source(access_token, api_host, series_list):
-    for series in series_list:
+def rank_series_by_source(access_token, api_host, selections_list):
```

Comment:
Could you please add a description and input/output ?

Reply:
Little confusing, but the docstring lives in `client.py`.

`client.py` docstrings are what are automatically turned into the API reference entries, like: https://developers.gro-intelligence.com/api.html#groclient.GroClient.rank_series_by_source 

The only difference between the lib version and the client version is that access_token and api_host are parameters instead of the client.py version uses the class' attribute. Keeping near-identical docstrings in both locations runs the risk of one being out-of-sync and causing more confusion, so we just refer to the client.py one as a single source of truth